app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html

```diff
@@ -85,6 +85,9 @@
 }
 
   @formHelper(action = controllers.routes.SubsidyController.postClaimDate) {
+<legend class="govuk-fieldset__legend--xl">
```

Comment:
I don't think the `legend` element is required here

Reply:
The legend is needed here for the styling of the header